packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -384,6 +384,7 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
 
   final RouteSettings settings;
 
+  /// The closest instance of this class that encloses the given context.
   static ModalRoute of(BuildContext context) {
```

Comment:
Yeah, nested navigators make it especially tricky to describe.  Maybe we should read "enclosing" to not refer strictly to the parent relation among widgets?  Maybe there's a better word to use that indicates that his one is different.